public/app/percona/settings/components/PrometheusConfiguration/PrometheusConfiguration.messages.ts

```diff
@@ -0,0 +1,9 @@
+export const Messages = {
+  description:
+    'Prometheus configuration offers a variety of options to help you fine-tune your monitoring setup, including modifying the scrape intervals, adjusting alerting rules, and more. Simply make your desired changes in the provided textfield and click "Submit changes" to store those changes to the base file. If you need help setting this up you can ',
```

Comment:
Is `textfield` the copy we were given? Or should it be `text field`?

Reply:
Wording will be revised, just haven't received updated content yet.